mlflow/utils/rest_utils.py

```diff
@@ -77,6 +95,37 @@ def http_request(
     Returns:
         requests.Response object.
     """
+    if host_creds.databricks_workspace_client:
+        try:
+            if method == "GET":
+                query = kwargs.get("params") or kwargs.get("json")
+                extra_kwargs = {"query": query}
+            else:
+                extra_kwargs = {"body": kwargs.get("json")}
+
+            raw_response = host_creds.databricks_workspace_client.api_client.do(
+                method=method,
+                path=endpoint,
+                headers=extra_headers,
+                raw=True,
+                **extra_kwargs,
+            )
+            return raw_response["contents"]._response
+        except Exception as e:
```

Comment:
why do we need to catch any exception?

Reply:
It wraps original exception with MlflowException and re-throw it:
```
            raise MlflowException(
                f"API request to endpoint '{endpoint}' failed with exception {e}"
            ) from e
```